Source/WebCore/inspector/agents/InspectorDOMAgent.cpp

```diff
@@ -180,11 +178,10 @@ static Color parseOptionalConfigColor(const String& fieldName, JSON::Object& con
 
 static bool parseQuad(Ref<JSON::Array>&& quadArray, FloatQuad* quad)
 {
-    const size_t coordinatesInQuad = 8;
-    double coordinates[coordinatesInQuad];
-    if (quadArray->length() != coordinatesInQuad)
+    std::array<double, 8> coordinates { };
```

Comment:
We don't need the `{ }`, There is no need to zero initialize the items as we're about to set them. Seems unnecessarily inefficient.

Reply:
Didn't see your comment before it got merged. I will address it in a follow-up.